example/src/subdir/ignored.js

```diff
@@ -0,0 +1 @@
+console.log("This should be lint error");
```

Comment:
Add a comment that we don't expect the error will be reported, because it appears in the .gitattributes (add the backlink)